extensions/typescript-language-features/src/commands/tsserverRequests.ts

```diff
@@ -17,8 +27,24 @@ export class TSServerRequestCommand implements Command {
 	) { }
 
 	public async execute(command: keyof TypeScriptRequests, args?: any, config?: any): Promise<unknown> {
```

Comment:
Instead of getting the `token` from the `args`, can we take it as an optional last parameter?

Reply:
Add an additional param